### ISSUE TITLE
 In high contrast Aquatic/desert modes, focus indicator is not visible on Fluent Auto complete control

### ISSUE THREAD

**Issue by PraleshGitHub** [microsoft/fluentui-blazor#2013]
# 🐛 Bug Report

 In high contrast Aquatic/desert modes, focus indicator is not visible on Fluent Auto complete control

## 💻 Repro or Code Sample

1. Go to https://www.fluentui-blazor.net/Autocomplete
2. Enable Aquatic/Desert color contrast mode from windows setting.
3. Click on the fluent autocomplete, a list will be populated.
4. Now use the down arrow key to chose the element
## 🤔 Expected Behavior

In High contrast Aquatic/Desert Modes, focus indicator should be clearly visible.

## 😯 Current Behavior

 In high contrast Aquatic/desert modes, focus indicator is not visible on Fluent Auto complete control.
In Desert mode, on hover on element populated in list is not visible.

## 🌍 Your Environment

* OS & Device: [Windows] on [PC]
* Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
* .NET 6 and Fluent UI Blazor library Version [3.7]

**Comment by dvoituron** [comment]
Will be fixed in the next release #2018
